### ISSUE TITLE
D-Bus name org.freedesktop.Notifications aquired even though notifications are off 

### ISSUE THREAD

**Issue by opatut** [linuxmint/cinnamon#7824]
```
 * Cinnamon 3.8.8
 * Distribution: Arch
 ```

**Issue**

When disabling Notifications via cinnamon-settings, the cinnamon root process still aquires the dbus name `org.freedesktop.Notifications`, making it impossible for a different daemon to aquire this name, so one cannot substitute the builtin notifications daemon with another one, such as [notify-osd](https://wiki.ubuntu.com/NotifyOSD) or [dunst](https://github.com/dunst-project/dunst).

Here is where the name is aquired, regardless of the notification enabled/disabled setting:

https://github.com/linuxmint/Cinnamon/blob/2499b0920ddbc86692f89cb54777486005add497/src/main.c#L143-L144

**Steps to reproduce**
* Disable notifications in cinnamon-settings
* Restart cinnamon
* Try to run another notification daemon
* `Cannot acquire 'org.freedesktop.Notifications': Name is acquired by PID '<cinnamon process>'.`

**Expected behaviour**
* Other daemon can aquire `org.freedesktop.Notficiations` name

**Comment by icarter09** [comment]
@opatut are you still experiencing this issue?

**Comment by opatut** [comment]
I am not entirely sure if the problem has been fixed, since it was a long time ago and I don't remember what I did with my system afterwards ;) But I am still running cinnamon underneath i3 and my dunst nowadays works just as expected. 

Therefore I believe this problem to be resolved. I'd say close the ticket, if it comes up again a new one can be opened.

**Comment by MegaphoneJon** [comment]
I'm experiencing this on Cinnamon 4.4.8.  Was experiencing on 4.0.x series and upgraded, but no joy.  Should this ticket be reopened, or shall I open a new one?

**Comment by MegaphoneJon** [comment]
Thanks for reopening @icarter09.  The replication steps above are still valid; let me know if there's anything I can produce (logs etc.) to assist.

**Comment by icarter09** [comment]
No problem @MegaphoneJon. Thanks for the update that the steps are still valid. Logs from `/home/<username>/.xsession-errors` and  `/var/log/syslog` would be helpful. Thanks again.

**Comment by mstyp** [comment]
> No problem @MegaphoneJon. Thanks for the update that the steps are still valid. Logs from `/home/<username>/.xsession-errors` and `/var/log/syslog` would be helpful. Thanks again.

I am not @MegaphoneJon, but I am having the same issue. Debian 10, cinnimon version 3.8, dont have any xsession logs and there does not seem to be anything usefull in /var/log/syslog.

**Comment by araujorm** [comment]
Confirmed still to be an issue in 2021, preventing using alternatives to notification display like dunst with cinnamon. Any ETA to solve this?

**Comment by almir-campos** [comment]
After spending a couple of hours trying to make dunst works on cinnamon, I just bumped into this issue thread. In my perspective its appalling that we are not free to use another notification daemon. I have no other alternative than get rid of cinnamon right away. It's sad because I was liking it very much.

**Comment by mtwebster** [comment]
dunst is unable to replace cinnamon during an existing session, but it *did* work at login - maybe a race condition, and not consistent.

**Comment by dechimp** [comment]
Having this issue still.

[cinnamon --version] Cinnamon 5.4.3
[uname -r] 5.18.12-3-MANJARO

On linux mint (which installs dunst 1.4.1 through apt), dunst seems to work fine for the most part.  On manjaro which installs dunst 1.9.3, I get the `name is acquired` message.

I tried compiling dunst 1.4.1 myself on manjaro but no luck.  I also tried removing `/usr/share/dbus-1/services/org.freedesktop.Notifications.service` but that also didnt work.

Does anyone have even have a workaround?
